README.md

```diff
@@ -28,12 +30,20 @@ An `nvmrc` file is included if using [nvm](https://github.com/creationix/nvm).
 
 ## Development
 
+From within the root directory:
+
+> npm run react-dev
+
+## Seed Database
+
+From within the root directory:
+
+> npm run seed
+
 ### Installing Dependencies
 
 From within the root directory:
 
-```sh
-npm install -g webpack
-npm install
-```
+> npm install -g webpack
```

Comment:
@VALE46GP Do not globally install modules. Additionally webpack should be installed locally as part of your dev-dependencies. You can either run one of your build scripts or npx webpack.